libphal/libphal.H

```diff
@@ -86,14 +87,15 @@ bool isDumpAllowed(struct pdbg_target *proc);
  * Exceptions: SbeError for pdbg lib function failure
  *             std::runtime_error - for file operation failure.
  */
-sbeError_t capturePOZFFDC(struct pdbg_target *target);
+sbeError_t capturePOZFFDC(struct pdbg_target *target, bool coSuccess = true);
```

Comment:
Better to avoid default argument if not necessitating changes in huge places. Also better to mark this as const